microceph/ceph/config.go

```diff
@@ -93,20 +104,24 @@ type ConfigDumpItem struct {
 }
 type ConfigDump []ConfigDumpItem
 
-func SetConfigItem(c types.Config) error {
-	configTable := GetConstConfigTable()
+func SetConfigItem(c types.Config, bypassChecks bool) error {
```

Comment:
From a separation of concerns POV I would suggest to -- instead of adding a flag -- split this function up, something like:

```
func SetConfigItemUnsafe(...) {
 // construct args, run ceph command
}

func SetConfigItem(...) {
  // perform checks
  if !canSet && !bypassChecks { 
...
  }

  SetConfigItemUnsafe(...)
}
```

I feel this clarifies the intent and is less error-prone wrt to passing in the flag.

Call sites of these functions would need to be updated too of course.

Reply:
I agree with @sabaini, using flags is prone to error, more so from the caller side.

Reply:
Ack.